3rd-party/woocommerce-stats.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+class WC_Stats {
+	/**
+	 * @var Jetpack
+	 **/
+	private $jetpack;
+
+	/**
+	 * @var WC_Stats
+	 **/
+	private static $instance = null;
+
+	static function init() {
+		// Tracking only Site pages
+		if ( is_admin() ) {
+			return;
+		}
+		// Make sure Jetpack is installed and active
+		if ( ! Jetpack::is_active() ) {
+			return;
+		}
+		// Make sure WooCommerce is installed and active
+		if ( ! in_array( 'woocommerce/woocommerce.php', apply_filters( 'active_plugins', get_option( 'active_plugins' ) ) ) ) {
```

Comment:
Is this an existing hook in Core? If so, could you add a docblock, like we do here for example?
https://github.com/Automattic/jetpack/blob/master/modules/carousel/jetpack-carousel.php#L237